package.json

```diff
@@ -50,7 +50,7 @@
     "react-dom": "0.14 - 16"
   },
   "dependencies": {
-    "signature_pad": "^2.3.2",
+    "signature_pad": "^4.0.0",
```

Comment:
This upgrade is a **_breaking_** change, but this PR does not make any other changes outside of a version upgrade.

Per the changelog, they are [incompatible versions](https://github.com/szimek/signature_pad/blob/master/CHANGELOG.md#400), however, as the SemVer major tag change suggests.
The `EventEmitter` change is breaking and affects the [`onEnd` and `onBegin` callbacks](https://github.com/agilgur5/react-signature-canvas#props) in this library/component. Those are currently in idiomatic React and don't work as smoothly/straightforward against `addEventListener` etc APIs. Will need to think of the optimal way to implement that, or require direct `signature_pad` usage or something, but at least one other internal change _must_ be made in addition to the upgrade.

It may very likely require a major version change of `react-signature-canvas` if our API has to change in order to accommodate the `EventEmitter` upstream change. TBD on that